README.md

```diff
@@ -1,168 +1,297 @@
-
-If you find and bug or have a feature request, please raise the issue on [GitHub Issues](../../issues). 
-
-## Contribute
+## Using MSAL
 
-We enthusiastically welcome contributions and feedback. You should clone the repo and start contributing now. 
+- Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application. To register your app, use [the Azure portal](https://aka.ms/AppRegistrationsPreview). For Azure AD B2C, checkout [how to register your app with B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-app-registration).  
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+### Requirements
 
-## Security Library
+- Android API Level 16+
 
-This library controls how users sign-in and access services. We recommend you always take the latest version of our library in your app when you can. We use [semantic versioning](http://semver.org) so you can control the risk of updating your app. For example, always downloading the latest minor version number (e.g. x.*y*.x) ensures you get the latest security and feature enhanements with the assurance that our API surface area has not changed. You can always see the latest version and release notes under the **Releases** tab of GitHub.
+### Step 1: Declare dependency on MSAL
 
-## Security Reporting
+Add to your app's build.gradle:
 
-If you find a security issue with our libraries or services, please report the issue to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as you can provide. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to **GitHub Issues** or any other public site. We will contact you shortly after receiving your issue report. We encourage you to get new security incident notifications by visiting [Microsoft technical security notifications](https://technet.microsoft.com/en-us/security/dd252948) to subscribe to Security Advisory Alerts.
+```gradle
+dependencies {
+    implementation 'com.microsoft.identity.client:msal:1.0.+'
+}
+```
 
+### Step 2: Create your MSAL configuration file
 
-Copyright (c) Microsoft Corporation.  All rights reserved. Licensed under the MIT License (the "License");
+>NOTE: Add link to configuration file documentation here. HELP, where is this?
```

Comment:
The config file itself is documented here: https://docs.microsoft.com/azure/active-directory/develop/msal-configuration
We have some how-to stuff for configuring an app, such as the quickstart.